### PR TITLE
Add EUS upgrade for azure-upi

### PR DESCRIPTION
**Pull request by liangxia**
Add EUS upgrade for azure-upi and resovle issue in https://github.com/openshift/release/pull/26194/files#r804329144

### CODE REVIEW

**Comment by liangxia** [comment]
/cc @wzheng1 @wsun1 
as they are interested.

**Review by pruan-rht — COMMENTED** [review]
/lgtm

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__nightly-4.10-upgrade-from-eus-4.8.yaml [hunk 1]
```diff
@@ -9,6 +22,10 @@ releases:
       stream: nightly
       version: "4.9"
   latest:
+    release:
```
Comment (jhou1) [inline ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__nightly-4.10-upgrade-from-eus-4.8.yaml:25]: I think we need a comment here explaining why for future participants to understand the workaround.

Comment (liangxia) [reply ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__nightly-4.10-upgrade-from-eus-4.8.yaml:25]: @jhou1  Unfortunately, we are not allowed to add comments in this yaml file. `make update` will remove all the comment lines.

**Pull request merged**